app/translations/en.json

```diff
@@ -1,11 +1,13 @@
 {
   "zingo": "Zingo!",
-  "version": "v0.0.75",
+  "version": "v0.0.80",
```

Comment:
Why 75 -> 80?  Why not 76?

Reply:
Well, if you compare between commits you can see the history... from 75->76->77......